back/src/graphql/index.ts

```diff
@@ -23,6 +23,7 @@ function formatError(err: GraphQLError): Error {
     return new UserInputError(originalError.message);
   }
 
+  console.error('Internal server error:', originalError);
```

Comment:
do you need to keep this?

Reply:
Yes ! It ads logs in case of an unknown error